backend/src/akvo/lumen/lib/transformation.clj

```diff
@@ -51,7 +51,7 @@
             :transformation (engine/execute-transformation tx-deps dataset-id job-execution-id (:transformation command))
             :undo (engine/execute-undo tx-deps dataset-id job-execution-id)))
         (db.job-execution/update-successful-job-execution tx-conn {:id job-execution-id}))
-      (let [dsv (db.transformation/latest-dataset-version-by-dataset-id tenant-conn {:dataset-id dataset-id})]
+      (doseq [dsv (db.transformation/latest-dataset-version-by-dataset-id tenant-conn {:dataset-id dataset-id})]
```

Comment:
theoretically, we only need to vacuum the table that we update not all of the dataset-version tables

Reply:
WIP https://github.com/akvo/akvo-lumen/pull/2870/commits/bde31eb21ed333fb537cc44ee01695b8e013df6f